src/definitions/modules/tab.js

```diff
@@ -364,8 +356,7 @@
                             settings.onLoad.call($tab[0], currentPath, parameterArray, historyEvent);
                         } else if (tabPath.search('/') === -1 && tabPath !== '') {
                             // look for in page anchor
-                            tabPath = module.escape.string(tabPath);
-                            $anchor = $('#' + tabPath + ', a[name="' + tabPath + '"]');
+                            $anchor = $('#' + tabPath + ', a[name="' + CSS.escape(tabPath) + '"]');
```

Comment:
The first part of the selector is not escaped
```suggestion
                            $anchor = $('#' + CSS.escape(tabPath) + ', a[name="' + CSS.escape(tabPath) + '"]');
```

Reply:
I fixed it and checked all other `#' \+` cases.